### ISSUE TITLE
What is the order of the Clenshaw Curtis rule?

### ISSUE THREAD

**Issue by sloisel** [JuliaGNI/QuadratureRules.jl#8]
This is very confusing:

```
julia> using QuadratureRules; ClenshawCurtisQuadrature(Float64,5).order
25
```

What is the meaning of the `order` field of the `QuadratureRule` object?

**Comment by michakraus** [comment]
Thanks for spotting (and reporting) this. This is clearly a bug in the code. I will fix this.

`order` should always refer to the polynomial exactness degree + 1. For Clenshaw-Curtis this would be just `5` in the example above.

Note that in the case of Clenshaw-Curtis the exactness degree/order is not very indicative of accuracy.

**Comment by michakraus** [comment]
Should be fixed in v0.1.6.
